pkg/utils/kube/image_test.go

```diff
@@ -58,9 +59,61 @@ func Test_extractImageInfo(t *testing.T) {
 				},
 			},
 		},
+		{
+			extractionConfig: `{"Task": [{"name": "steps", "fields": ["spec", "steps", "*"], "value": "image"}]}`,
```

Comment:
Some examples of extractionConfigs and their application on custom resources like tekton tasks.